### PR TITLE
Feat: Power Consumed Watts functionality added (barebones)

### PR DESCRIPTION
**Pull request by QuantumPosix**

### CODE REVIEW

**Comment by QuantumPosix** [comment]
fixes #309

File: src/badfish/main.py [hunk 1]
```diff
@@ -74,6 +74,7 @@ async def init(self):
         self.manager_resource = await self.find_managers_resource()
         self.bios_uri = "%s/Bios/Settings" % self.system_resource[len(self.redfish_uri) :]
 
+
```
Comment (grafuls) [inline src/badfish/main.py:77]: remove extra line

**Review by grafuls — CHANGES_REQUESTED** [review]
Include tests

**Review by sadsfae — APPROVED** [review]
Nice beginnings of power awareness with Badfish.  I'd expect to be able to build on this to create polling intervals and do the backend math on average / high / low over a reasonable polling period.

**Pull request merged**